app/views/search/index.html.erb

```diff
@@ -8,12 +8,10 @@
   </div>
 </div>
 
-<% @results.sorted.each do |result| %>
-  <% if result.class == Answer %>
+<% @response.results.each do |result| %>
+  <% if result.class.name == 'Answer' %>
```

Comment:
This is important for how we think about checking these results. We can't check the class directly because it isn't _really_ a class. It's an ActiveRecord Model.